tests/manage/mcg/test_non_ocs_taint_and_toleration.py

```diff
@@ -0,0 +1,123 @@
+import logging
+import pytest
+import time
+
+from ocs_ci.ocs import ocp, constants, defaults
+
+from ocs_ci.framework.testlib import (
+    E2ETest,
+    ignore_leftovers,
+    skipif_tainted_nodes,
+    skipif_managed_service,
+)
+from ocs_ci.ocs.resources.pod import (
+    get_all_pods,
+    wait_for_pods_to_be_running,
+    check_toleration_on_pods,
+)
+from ocs_ci.ocs.node import (
+    taint_nodes,
+    untaint_nodes,
+    get_worker_nodes,
+)
+from ocs_ci.framework.pytest_customization.marks import bugzilla
+from ocs_ci.helpers.sanity_helpers import Sanity
+
+logger = logging.getLogger(__name__)
+
+
+@ignore_leftovers
+@skipif_tainted_nodes
+@skipif_managed_service
+@bugzilla("2115613")
+class TestNonOCSTaintAndTolerations(E2ETest):
```

Comment:
we already have test_non_ocs_taint_and_toleration.py  which covers all these steps. 
Can you check if you adjust the existing test to handle mcg-only by adding the relevant conditions?